Classes/Hooks/Datahandler/Database.php

```diff
@@ -103,11 +86,29 @@ public function fetchOneTranslatedRecord(int $uid, int $language): ?array
         return $record;
     }
 
-    /**
-     * @param int $parent
-     * @param int $language
-     * @return array
-     */
+    public function fetchOneTranslatedRecordByl18nParent(int $uid, int $language): ?array
```

Comment:
Please rename to "fetchOneTranslatedRecordByLocalizationParent" or "fetchOneTranslatedRecordByL10Parent"